src/tests/end-to-end/tests/details-view/__snapshots__/tabstop.test.ts.snap

```diff
@@ -0,0 +1,201 @@
+  <div
+    id="insights-shadow-container"
+  >
+    <link
+      href="{{EXTENSION_URL_BASE}}/injected/styles/default/injected.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="{{EXTENSION_URL_BASE}}/bundle/injected.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <div
+      class="insights-container insights-highlight-container insights-tab-stops"
+    >
+      <svg
+        height="{{ENVIRONMENT_SENSITIVE_POSITION}}px"
+        width="{{ENVIRONMENT_SENSITIVE_POSITION}}px"
+      >
+        <defs>
+          <filter
+            filterUnits="userSpaceOnUse"
+            id="insights-tab-stops-solid-shadow"
+          >
+            <femorphology
+              in="SourceGraphic"
+              operator="dilate"
+              radius="1"
+              result="expand"
+            />
+            <feoffset
+              dx="1"
+              dy="0"
+              in="expand"
+              result="shadow_1"
+            />
+            <feoffset
+              dx="-1"
+              dy="0"
+              in="expand"
+              result="shadow_2"
+            />
+            <feoffset
+              dx="0"
+              dy="1"
+              in="expand"
+              result="shadow_3"
+            />
+            <feoffset
+              dx="0"
+              dy="-1"
+              in="expand"
+              result="shadow_4"
+            />
+            <femerge
+              result="shadow"
+            >
+              <femergenode
+                in="shadow_1"
+              />
+              <femergenode
+                in="shadow_2"
+              />
+              <femergenode
+                in="shadow_3"
+              />
+              <femergenode
+                in="shadow_4"
+              />
+              <femergenode
+                in="expand"
+              />
+            </femerge>
+            <feflood
+              flood-color="white"
+            />
+            <fecomposite
+              in2="shadow"
+              operator="in"
+              result="shadow"
+            />
+            <femerge>
+              <femergenode
+                in="shadow"
+              />
+              <femergenode
+                in="SourceGraphic"
+              />
+            </femerge>
+          </filter>
+        </defs>
+      </svg>
+    </div>
+  </div>
+</DocumentFragment>
+`;
+
+exports[`Tabstop tests tabstop from fastpass if visualHelper is turned on after starting tabstop and pressing tabs on target page 1`] = `
+<DocumentFragment>
+  <div
+    id="insights-shadow-container"
+  >
+    <link
+      href="{{EXTENSION_URL_BASE}}/injected/styles/default/injected.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="{{EXTENSION_URL_BASE}}/bundle/injected.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <div
+      class="insights-container insights-highlight-container insights-tab-stops"
+    >
+      <svg
```

Comment:
That the <svg> element here contains no `<text>`, `<line>`, or `<ellipse>` elements suggests very strongly that these tests aren't validating what I suspect you intended.

Reply:
Tried to do this, but now i need to write more functions to sanitize the visualizations!